src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String category;
+
+    public Task(String description) {
+        this.description = description;
+        this.category = "";
```

Comment:
Similar issue observed in Event as well as Deadline

Reply:
Thanks for your feedback !